datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/models.py

```diff
@@ -133,17 +135,23 @@ def models(ctx, check, sync, verbose):
                 else:
                     files_failed[model_file_path] = True
                     check_display_queue.append(
-                        lambda model_file=model_file, **kwargs: echo_failure(
-                            f'File `{model_file}` is not in sync, run "ddev validate models -s"', **kwargs
+                        (
+                            echo_failure,
+                            f'File `{model_file}` is not in sync, run "ddev validate models -s"',
                         )
                     )
 
-        if check_display_queue or verbose:
+            if not check_display_queue and verbose:
```

Comment:
Why the change in logic?

Reply:
annotations are annotated on each file. The previous display_queue was vague. I restructured the display queue to a map of file_path to display_queue of errors occurred in the nested for loop.

Also the verbose line was erroneous. It was appending `Valid spec` anytime the `-v` flag was used, instead of actually checking if there were errors and if it's actually valid.